app/src/main/res/layout/download_list_item.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/headerTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="14sp"
+        android:textColor="?primaryText"
+        android:fontFamily="@font/metropolis_semibold"
+        android:paddingStart="20dp"
+        android:paddingEnd="0dp"
+        android:layout_marginBottom="8dp"
+        tools:text="Downloads"
+        android:visibility="visible" />
+
+    <TextView
+        android:id="@+id/contentTextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="14sp"
+        android:textColor="?primaryText"
+        android:fontFamily="@font/metropolis_semibold"
+        android:paddingStart="20dp"
+        android:paddingEnd="10dp"
+        android:layout_marginBottom="8dp"
+        style="@style/TextAppearance.AppCompat.Subhead.Inverse"
+        android:layout_gravity="bottom"
+        android:paddingTop="20dp"
+        android:textAppearance="?attr/textAppearanceListItem"/>
+
+    <TextView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="12dp"
+    android:ellipsize="end"
+    android:singleLine="true"
+    android:textAlignment="viewStart"
+    android:textColor="?secondaryText"
+    android:textSize="12sp"
+    android:id="@+id/sourceTextView"
+    android:textAppearance="?attr/textAppearanceListItem"
+        android:paddingBottom="20dp" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="12dp"
+        android:ellipsize="end"
+        android:singleLine="true"
+        android:textAlignment="viewStart"
+        android:textColor="?secondaryText"
+        android:textSize="12sp"
+        android:id="@+id/sizeTextView"
+        android:textAppearance="?attr/textAppearanceListItem"/>
+
+    <View
+        android:id="@+id/separator"
```

Comment:
If we need a divider we could have our own `DividerItemDecoration` set for the RecycleView, one that would easily know to not be set after the last item.